recipes/tinycthreadpool/all/conanfile.py

```diff
@@ -0,0 +1,53 @@
+import os
+
+from conan import ConanFile
+from conans import CMake, tools
+from conan.tools.files import get
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.scm import Version
+
+required_conan_version = ">=1.53.0"
+
+
+class TinyCThreadPoolConan(ConanFile):
+    name = "tinycthreadpool"
+    description = "Portable C thread pool"
+    license = "BSD-2-Clause"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/bennyhuo/tinycthreadpool"
+    topics = ("thread-pool", "threading", "pure-c")
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "CMakeDeps"
+    
+    def configure(self):
+        self.settings.rm_safe("compiler.libcxx")
+        self.settings.rm_safe("compiler.cppstd")
+
```

Comment:
```suggestion

    def validate(self):
        check_min_vs(self, "1921")
        if is_msvc_static_runtime(self):
            raise ConanInvalidConfiguration("{self.ref} does not support static runtime.")

```
We have new and powerful tool for Microsoft https://docs.conan.io/en/latest/reference/conanfile/tools/microsoft.html